WalletWasabi/Services/WalletService.cs

```diff
@@ -1137,101 +1137,79 @@ public Operation(Script script, Money amount, string label)
 			// 4. Get and calculate fee
 			Logger.LogInfo<WalletService>("Calculating dynamic transaction fee...");
 
-			FeeRate feePerBytes = (feeTarget.HasValue) ? Synchronizer.GetFeeRate(feeTarget.Value) : feeRate;
+			FeeRate feePerBytes = (feeTarget is int v) ? Synchronizer.GetFeeRate(v) : feeRate;
+			feePerBytes = FeeRate.Max(feePerBytes, new FeeRate(1m));
 
-			feePerBytes = feePerBytes.SatoshiPerByte < 1 ? new FeeRate(1m) : feePerBytes; // Use the sanity check that under 2 satoshi per bytes should not be displayed. To correct possible rounding errors.
-
-			bool spendAll = spendAllCount == 1;
-			int inNum;
-			if (spendAll)
-			{
-				inNum = allowedSmartCoinInputs.Count;
-			}
-			else
-			{
-				int expectedMinTxSize = 1 * Constants.P2wpkhInputSizeInBytes + 1 * Constants.OutputSizeInBytes + 10;
-				inNum = SelectCoinsToSpend(allowedSmartCoinInputs, toSend.Select(x => x.Amount).Sum() + feePerBytes.GetTotalFee(expectedMinTxSize)).Count();
-			}
-
-			// https://bitcoincore.org/en/segwit_wallet_dev/#transaction-fee-estimation
-			// https://bitcoin.stackexchange.com/a/46379/26859
-			int outNum = spendAll ? toSend.Length : toSend.Length + 1; // number of addresses to send + 1 for change
-			int vSize = NBitcoinHelpers.CalculateVsizeAssumeSegwit(inNum, outNum);
-			Logger.LogInfo<WalletService>($"Estimated tx size: {vSize} vbytes.");
-
-			// Multiply the standard by 1.1 to avoid possible off by one errors in flawed implementations.
-			var minFeeRate = new FeeRate(1.1m * new StandardTransactionPolicy().MinRelayTxFee.SatoshiPerByte);
-			if (feePerBytes < minFeeRate)
-			{
-				feePerBytes = minFeeRate;
-			}
-
-			Money fee = feePerBytes.GetTotalFee(vSize);
-
-			Logger.LogInfo<WalletService>($"Fee: {fee.Satoshi} Satoshi.");
```

Comment:
Can we still log the paid fee? IMO it is useful and we really don't know whether someone is using it or not.

Reply:
We still do `Logger.LogInfo<WalletService>($"Fee: {fee.Satoshi} Satoshi.");`